docs/api-reference/full-text/index.mdx

```diff
@@ -363,6 +363,30 @@ The following tokenizer names are accepted by the `paradedb.tokenizer` function.
   Equivalent to `stem` with language set to `English`.
 </ParamField>
 
+#### Picking a Tokenizer
+
+If you're not sure which tokenizer to use, you can refer to `paradedb.tokenizer` for a list of all available tokenizers. To evaluate and compare different tokenization strategies, use `paradedb.tokenize`.
```

Comment:
`paradedb.tokenizer` is missing an s, I think

Reply:
I don't know if we have a documentation style guide, so I'll just throw this out there...

It's generally a good idea for documentation to avoid talking directly to the reader.  With that in mind, I might rewrite this paragraph as

```
The `paradedb.tokenizers()` function returns a list of all available tokenizer names.  Evaluation of any named tokenizer is accomplished with the `paradedb.tokenize()` function.
```

or something to that effect.

¯\_(ツ)_/¯

Reply:
I like that!